packages/client/hmi-client/src/services/calibrate-workflow.ts

```diff
@@ -108,3 +115,28 @@ export const renderLossGraph = (
 	}
 	yAxisGroup.attr('transform', `translate(${marginLeft}, 0)`).call(yAxis);
 };
+
+export const autoCalibrationMapping = async (modelOptions: State[], datasetOptions: any[]) => {
+	const result = [] as CalibrateMap[];
+	modelOptions.forEach((modelOption) => {
```

Comment:
I know everytime i write a for loop people ask me to use forEach instead but I cannot use `break` in this. which means when I find a correct result it keeps chugging away 
Will yall complain if i write a for loop for this instead of a forEach?

Reply:
There's nothing wrong with a `for (....)` construct:
- If it makes sense to use it, use it.
- If it makes the code easier to comprehend, use it.